src/chainparams.cpp

```diff
@@ -474,14 +474,51 @@ class CRegTestParams : public CChainParams {
     /**
      * Allows modifying the Version Bits regtest parameters.
      */
-    void UpdateVersionBitsParameters(Consensus::DeploymentPos d, int64_t nStartTime, int64_t nTimeout)
+    void UpdateVersionBitsParameters(Consensus::DeploymentPos d, int64_t startheight, int64_t timeoutheight)
     {
-        consensus.vDeployments[d].nStartTime = nStartTime;
-        consensus.vDeployments[d].nTimeout = nTimeout;
+        consensus.vDeployments[d].startheight = startheight;
+        consensus.vDeployments[d].timeoutheight = timeoutheight;
```

Comment:
I know this is just regtest, but an integral type of size 64-bit will be truncated if assigned to an integral type of size 32-bit

Reply:
Maybe have CheckVBitsHeights check this